cob_camera_sensors/CMakeLists.txt

```diff
@@ -15,7 +15,7 @@ generate_messages(
 )
 
```

Comment:
What about the [Boost depends](https://github.com/ipa320/cob_driver/blob/indigo_dev/cob_camera_sensors/ros/src/tof_camera.cpp#L80)?